cmd/server/wire.go

```diff
@@ -37,31 +37,31 @@ type Application struct {
 	common.GlobalInitializer
 	common.Migrator
 
-	App                     app.Service
-	AppStripe               appstripe.Service
-	AppSandboxProvisioner   common.AppSandboxProvisioner
-	Customer                customer.Service
-	Billing                 billing.Service
-	EntClient               *db.Client
-	EventPublisher          eventbus.Publisher
-	EntitlementRegistry     *registry.Entitlement
-	FeatureConnector        feature.FeatureConnector
-	IngestCollector         ingest.Collector
-	KafkaProducer           *kafka.Producer
-	KafkaMetrics            *kafkametrics.Metrics
-	Logger                  *slog.Logger
-	MeterRepository         meter.Repository
-	NamespaceHandlers       []namespace.Handler
-	NamespaceManager        *namespace.Manager
-	Notification            notification.Service
-	Meter                   metric.Meter
-	Plan                    plan.Service
-	RouterHook              func(chi.Router)
-	Secret                  secret.Service
-	Subscription            common.SubscriptionServiceWithWorkflow
-	SubscriptionPlanAdapter plansubscription.Adapter
-	StreamingConnector      streaming.Connector
-	TelemetryServer         common.TelemetryServer
+	App                       app.Service
+	AppStripe                 appstripe.Service
+	AppSandboxProvisioner     common.AppSandboxProvisioner
+	Customer                  customer.Service
+	Billing                   billing.Service
+	EntClient                 *db.Client
+	EventPublisher            eventbus.Publisher
+	EntitlementRegistry       *registry.Entitlement
+	FeatureConnector          feature.FeatureConnector
+	IngestCollector           ingest.Collector
+	KafkaProducer             *kafka.Producer
+	KafkaMetrics              *kafkametrics.Metrics
+	Logger                    *slog.Logger
+	MeterRepository           meter.Repository
+	NamespaceHandlers         []namespace.Handler
+	NamespaceManager          *namespace.Manager
+	Notification              notification.Service
+	Meter                     metric.Meter
+	Plan                      plan.Service
+	RouterHook                func(chi.Router)
+	Secret                    secret.Service
+	Subscription              common.SubscriptionServiceWithWorkflow
+	SubscriptionChangeService plansubscription.PlanSubscriptionService
```

Comment:
Can we initialize this in the Subscription service? 
I don't think we need this granularity to be able to run one of them independently.